frontend/apps/docs/app/docs/sitemap.ts

```diff
@@ -0,0 +1,39 @@
+// biome-ignore lint/correctness/noNodejsModules: This file is used to generate the sitemap and is executed on the server-side only, requiring Node.js modules.
+import fs from 'node:fs'
+// biome-ignore lint/correctness/noNodejsModules: This file is used to generate the sitemap and is executed on the server-side only, requiring Node.js modules.
```

Comment:
Is it better to write this without using ``ignore``? I would appreciate any advice🙏!

Reply:
Ignoring seems to be fine.

The reason I enabled this in `packages/configs/biome.jsonc` was that some packages are designed to run in non-Node.js environments.

However, given cases like this one, it seems better to disable it by default and enable it individually in each package as needed. I will make the adjustments accordingly.

Reply:
Thank you🙏. I understand!
I'll keep it as is this time.

Reply:
> However, given cases like this one, it seems better to disable it by default and enable it individually in each package as needed. I will make the adjustments accordingly.

LGTM 👍🏻

Reply:
Issued: https://github.com/liam-hq/liam/issues/401